scripts/gulp/src/plugins/gulp-example-source.ts

```diff
@@ -12,13 +12,13 @@ import { ExampleSource } from './util/docs-types';
 
 const prettierConfig = require('../../../../prettier.config');
 
-// eslint-disable-next-line deprecation/deprecation
-const ESLint = new CLIEngine({
```

Comment:
[removed in v8](https://eslint.org/docs/latest/use/migrate-to-8.0.0#remove-cliengine)